README.md

```diff
@@ -3,7 +3,9 @@
 [![Promote charm](https://github.com/canonical/github-runner-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/github-runner-operator/actions/workflows/promote_charm.yaml)
 [![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
 
-This machine charm creates self-hosted runners for running GitHub Actions. Each unit of this charm will start a configurable number of OpenStack or LXD based virtual machines to host them. Every runner performs only one job, after which it unregisters from GitHub to ensure that each job runs in a clean environment.
+**Important** Use the [local-lxd](https://charmhub.io/github-runner?channel=local-lxd/stable) track for LXD runners.
```

Comment:
Do you know if there is documentation available for LXD if users click on this link? Otherwise it might be worth to add a sentence explaining what LXD runners are/were and that development is concentrating on Openstack for now.

Reply:
Yeah I don't think that Charmhub will update the docs based on a specific channel. I agree with adding another sentence